### ISSUE TITLE
Banned

### ISSUE THREAD

**Issue by biskit777** [SmartlyDressedGames/Unturned-3.x-Community#2500]
Hello, I immediately point out that this is my first and last post on GitHub

 and I am from Poland and I use translator, because it is too long message for my skills, and I just want to convey what I want to convey, so that it is clear, not necessarily syntactically correct. So I will start with a description of what happened, about 200 days ago I caught a strong tilt after getting raid from cheaters, because then they invited us to discord and showed the screen, they had cheats. I decided that my friends and I will check the protection of anticheat (on a new account), I entered with a shared screen, and checked some cheats from the Internet, I did not get banned so I continued to play, the server was empty, I shot zombies with aimbot and so on, after all the fun I removed the cheats and we decided that we will find a server with active administration, but here was the problem that my friends entered the server we found, but I could not, I was banned from multiplayer. I've been playing Unturned since 2015, I bought the gold package and so on, but I just uhh, got upset and so it came out, I got banned on each of my accounts, most likely it's a computer ban because even though I removed the cheats and so on back then, I still get banned on new accounts. I know I already have one vac ban on my account, it was caused by using a site to receive sms from steam, a free phone number or something, a person who also used that site got banned, and all users of that site got vac right afterwards including me on a couple of accounts. Everything may look unbelievable, I take it into account, my ban is 100% right because I used cheats, but I regret that I got banned from a game I was with for a long time and it was my everyday life for some time, my cheating didn't destroy anyone's gameplay (except zombies), because all I did was shoot zombies on an empty server with aimbot on screenshare, I totally don't know why I did it, but I would like to ask for a second chance, I served almost 200 days of punishment, my friends ask me to do something about it because they want to play with me again, I invited them to this game, thanks to me they started playing unturned, the fact is that it's only 5 people, but still something. This game meant and means a lot to me, so I ask to be banned, although the ban is right, I think I deserve a second chance, but I'm not here to pass judgment on myself, I leave the opinion to the unturned community, as well as the creator of the game Nelson

My steam profile: https://steamcommunity.com/id/ValveGabe/

This is not my only profile, I had three more (forgotten passwords, but recovered), but this is the one I care about the most because it was my first profile, and I have the most hours on it, which adding up all my accounts is over a thousand.

In conclusion, the ban is right, I admit it 100%, but I ask for a second chance.

![obraz_2021-03-24_220706](https://user-images.githubusercontent.com/65959632/112383328-46544e00-8ced-11eb-945f-1ac99cece9c7.png)

**Comment by rube200** [comment]
Please don't make Nelson waste his time with this requests.
He is already busy doing netcode rewrite and U4

**Comment by SDGNelson** [comment]
Sorry to deny your request, but the ban is permanent.

**Comment by TH3AL3X** [comment]
> Please don't make Nelson waste his time with this requests.
> He is already busy doing netcode rewrite and U4

lmfao xd

**Comment by biskit777** [comment]
> Sorry to deny your request, but the ban is permanent.

can i atleast dont have hwid ban, and play on other account?

**Comment by educatalan02** [comment]
> > Sorry to deny your request, but the ban is permanent.
> 
> can i atleast dont have hwid ban, and play on other account?

SDGNelson commented 18 hours ago
Sorry to deny your request, but the ban is permanent.
